### DISCUSSION TITLE
Quebec data

### DISCUSSION THREAD

**Discussion by ArkadNaisir** [electricitymaps/electricitymaps-contrib#3209] [Ideas]
Hi, I am unsure what data is needed, but here is the production portfolio of Quebec, excluding extremely small hydro-electric dams in a few towns. https://www.hydroquebec.com/generation/generating-stations.html

**Comment by christbelanger** [comment]
Hi !

Timing is everyting :-)

We just published the info, I also write a discussion note about it : https://github.com/tmrowco/electricitymap-contrib/discussions/3210 

Christophe

**Comment by madsnedergaard** [comment]
Thanks for the info, let's keep all future discussions in https://github.com/tmrowco/electricitymap-contrib/issues/3218 :)
